spec/requests/vendors_spec.rb

```diff
@@ -5,16 +5,33 @@
   describe "searching for vendors" do
     before do
       VCR.use_cassette("search-vendors") do
-        get '/eholdings/jsonapi/vendors/?q=ebsco', headers: okapi_headers
+        get '/eholdings/jsonapi/vendors/?q=e', headers: okapi_headers
       end
     end
 
     let!(:json) { Map JSON.parse response.body }
 
     it "gets a list of resources" do
       expect(response).to have_http_status(200)
-      expect(json.data.length).to equal(2)
-      expect(json.meta.totalResults).to equal(2)
+      expect(json.data.length).to equal(25)
+      expect(json.meta.totalResults).to equal(101)
+    end
+
+    describe "with pagination" do
+      before do
+        VCR.use_cassette("search-vendors-page2") do
+          get '/eholdings/jsonapi/vendors/?q=e&page=2', headers: okapi_headers
+        end
+      end
+
+      let!(:json2) { Map JSON.parse response.body }
+
+      it "gets a different list of resources" do
```

Comment:
I don't think this test should be passing. `page => offset` transform doesn't seem to have been implemented for the `vendor` model. 🤔

Reply:
Yup, I should've been using `not_to eql` instead of `not_to equal`. (https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/equality-matchers)

Fixed.